### ISSUE TITLE
Rails/SaveBang should flag `update_attributes`

### ISSUE THREAD

**Issue by rhys-vdw** [bbatsov/rubocop#4100]
`update_attributes` is an alias for `update` and should trigger `Rails/SaveBang` under the same conditions. I think `:update_attributes` just needs to be added [here](https://github.com/bbatsov/rubocop/blob/04f49f64d63c6cd8149c729296a766be1b888d71/lib/rubocop/cop/rails/save_bang.rb#L49).

--------

## Expected behavior

`Rails/SaveBang` should error on `update_attributes` in the same conditions as `update` (as it is an alias).

## Actual behavior

It permitted it.

## Steps to reproduce the problem

Turn on `Rails/SaveBang` and lint against `model.update_attributes`.

## RuboCop version

```
-> rubocop -V
0.46.0 (using Parser 2.3.3.0, running on ruby 2.3.3 x86_64-darwin16)
```

**Comment by bbatsov** [comment]
The problem is that `update` is a super generic name and this will trigger A LOT of false positives in any codebase...

**Comment by Ana06** [comment]
@bbatsov 
> The problem is that update is a super generic name and this will trigger A LOT of false positives in any codebase...
But it is the other way around, it already complains about `update`, but it doesn't complain about `update_attributes`

**Comment by bbatsov** [comment]
LOL! Silly me! Thanks for pointing this out @Ana06! :-) Yeah, fixing this should be trivial then.
